features/plugin-install.feature

```diff
@@ -6,7 +6,7 @@ Feature: Install WordPress plugins
     When I run `wp plugin install https://github.com/runcommand/one-time-login/archive/master.zip --activate`
     Then STDOUT should contain:
       """
-      Downloading install package from https://github.com/runcommand/one-time-login/archive/master.zip
+      from https://github.com/runcommand/one-time-login/archive/master.zip
```

Comment:
Could we maybe add a `Then STDOUT should contain` step with `Downloading install` message as well here, and keep `package` in front of `from`? To emphasize that we expect a download....

Reply:
Ah, `installation` contains `install` ... right. 😄 
Thanks!